packages/visualizations/src/components/utils/ColorsLegend.svelte

```diff
@@ -45,84 +47,96 @@
     onDestroy(rotationDebounce.cancel);
 </script>
 
-<div class={`legend-colors legend-colors--${variant}`}>
-    {#if title}
-        <div class="legend-colors-title">{title}</div>
-    {/if}
-    {#if colorScale.type === 'gradient'}
-        <!-- Gradient color boxes, no custom labels, only displaying min and max -->
-        <div
-            class="legend-colors-color-box-gradient"
-            style="--legend-color:linear-gradient(to right, {colorScale.colors.start}, {colorScale
-                .colors.end})"
-        />
-        <div class="legend-colors-values">
-            <div>{defaultCompactLegendNumberFormat(dataBounds.min)}</div>
-            <div>{defaultCompactLegendNumberFormat(dataBounds.max)}</div>
-        </div>
-    {:else if colorScale.type === 'palette'}
-        <!-- Palette color boxes, row display, no labels only displaying palettes steps -->
-        <div class="legend-colors-container-palette" bind:clientWidth={legendWidth}>
-            <div class="legend-colors-row-color-box-palette">
-                {#each colorScale.colors as color}
-                    <div class="legend-colors-color-box-palette" style="--box-color: {color}" />
-                {/each}
-            </div>
+<div class={`legend-container legend-container--${position}`}>
```

Comment:
The extra wrapper div is not necessary… but it doesn't hurt either 🤷 Probably make things easier to stabilize in the future and that's really just one div…